src/core/include/openvino/op/constant.hpp

```diff
@@ -716,6 +716,7 @@ class OPENVINO_API Constant : public Op {
     element::Type m_element_type;
     Shape m_shape{};
     std::shared_ptr<ngraph::runtime::AlignedBuffer> m_data;
+    std::shared_ptr<ov::frontend::FrontEndManager> m_femgr;
```

Comment:
Actually, I only found op::constant will trigger segment fault during its destruction, so only put the dependency into op::constant. Ok let me check whether it can work if move to node:)

Reply:
Have applied void for it: std::shared_ptr<void> m_femgr;